common/test/acceptance/pages/lms/course_nav.py

```diff
@@ -178,6 +178,9 @@ def is_on_section(self, section_title, subsection_title):
         That's true right after we click the section/subsection, but not true in general
         (the user could go to a section, then expand another tab).
         """
+        # TODO: TNL-6546: This call is still needed, but it will need to be refactored
```

Comment:
As we just discussed, I prefer the idea of having top level page objects, so we would have CourseHomePage and CourseContentPage.